src/components/Todo/Todo.tsx

```diff
@@ -0,0 +1,132 @@
+import React, {
+  FC, useEffect, useRef, useState,
+} from 'react';
+import cn from 'classnames';
+import { Todo as TodoType } from '../../types/Todo';
+
+interface Props {
+  todo: TodoType,
+  deleteTodo: (todoId: number) => void,
+  loading: boolean,
```

Comment:
```suggestion
  isLoading: boolean,
```
Boolean variables should have prefix `is` or `has`